pandas/core/series.py

```diff
@@ -858,9 +858,14 @@ def ravel(self, order: str = "C") -> ArrayLike:
         Examples
         --------
         >>> s = pd.Series([1, 2, 3])
-        >>> s.ravel()
+        >>> s.ravel()  # doctest: +SKIP
         array([1, 2, 3])
         """
+        warnings.warn(
+            "Series.ravel is deprecated. Use numpy.ravel directly.",
+            FutureWarning,
+            stacklevel=find_stack_level(),
```

Comment:
```suggestion
            stacklevel=2,
```

For a simple case like this where the warning is directly raised in the top-level method, I prefer setting it explicitly, which avoids potential hickups with the function (I think we had perf issues in certain cases, and this also avoids getting a wrong stacklevel in case we forget a use case of ravel in our own code base)

Reply:
sure